packages/metastream-app/src/locale/en-US.ts

```diff
@@ -135,6 +136,12 @@ export default {
   theaterModeDesc:
     'Hide all non-video content on the webpage. Note that this might also hide soft subtitles.',
   thirdPartyIntegrations: 'Third-party Integrations',
+  tipsGettingStarted1: 'Tips for getting started',
```

Comment:
Two things to note here:
1. Lines that were broken into several lines can be merged into one string. CSS's layout engine will handle line wrapping for us.
2. It might be best for semantics to describe the locale keys without using numbers. For example:
```suggestion
  tipsGettingStartedHeader: 'Tips for getting started',
```